packages/sdk/src/browser-wallets/xverse/index.ts

```diff
@@ -25,10 +49,64 @@ function isInstalled() {
  * @returns An array of WalletAddress objects.
  */
 async function getAddresses(
-  _network: BrowserWalletNetwork = "mainnet",
-  _readOnly?: boolean,
+  network: BrowserWalletNetwork = "mainnet",
+  readOnly?: boolean,
 ): Promise<WalletAddress[]> {
-  return [];
+  if (!isInstalled()) {
+    throw new BrowserWalletNotInstalledError("Xverse not installed");
+  }
+
+  if (readOnly) {
+    throw new OrditSDKError("Read only mode is not supported on Xverse");
```

Comment:
@thedoublejay should we keep this here? or remove the parameter from this file's API

Reply:
remove it, it's not needed for Xverse

Reply:
Removed.